### PR TITLE
Random fixes

### PR DESCRIPTION
**Pull request by JRI98**

### CODE REVIEW

File: exercises/005_create_transfer.sh [hunk 1]
```diff
@@ -19,4 +19,4 @@ tb "create_transfers debit_account_id=500 credit_account_id=501 amount=100 ledge
 # But we wouldn't recommend doing this in production!
 
 # We recommend using TigerBeetle Time-Based Identifiers for most IDs:
-# https://docs.tigerbeetle.com/coding/data-modeling#id
```
Comment (emschwartz) [inline exercises/005_create_transfer.sh:22]: This section was renamed but we hadn't deployed the latest changes to the live docs site yet. It's being deployed now

**Review by emschwartz — COMMENTED** [review]
The links to the docs site should stay as is, but otherwise looks good. Thank you for spotting and fixing those issues!

File: exercises/005_create_transfer.sh (hunk 1 above)
Comment (JRI98) [reply exercises/005_create_transfer.sh]: Had not noticed it. Reverted

**Review by emschwartz — APPROVED** [review]

**Comment by JRI98** [comment]
Looks like patches broke https://github.com/tigerbeetle/tigerlings/actions/runs/9844327671/job/27177484070

**Comment by emschwartz** [comment]
Ah right, didn't wait for the action to complete before merging. I'll fix that along with the issue you opened

**Pull request merged**